pom.xml

```diff
@@ -192,4 +192,18 @@
 			<url>https://github.com/apimatic/core-interfaces-java/blob/main/LICENSE</url>
 		</license>
 	</licenses>
+
```

Comment:
No need to update the pom and push it to main rather the repository tag should be added during testing the core library or okhttp-client-adapter because they need core interfaces library while building the libraries. As these configuration should be added by the project that is consuming the snapshot releases.